Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBServiceTests/RemoteMongoClientIntTests.swift

```diff
@@ -15,8 +16,8 @@ class RemoteMongoClientIntTests: BaseStitchIntTestCocoaTouch {
     
     private lazy var mongodbUri: String = pList?[mongodbUriProp] as? String ?? "mongodb://localhost:26000"
 
-    private let dbName = ObjectId().description
-    private let collName = ObjectId().description
+    private let dbName = "dbName"
```

Comment:
[q] why did this change?

Reply:
The database name for sync ended up being too long and mongo was rejecting it.

Reply:
really? that seems odd. it seems like "db.collName" is allowed to be 120 bytes. Is something else making the name longer? is `.description` actually the hex value of the db? this could become a problem when we are namespacing by user id and client app id

Reply:
The description is just the odd. I think client app id is short enough that it shouldn't be a problem, but yes, user id may cause issues. We may want to rename the sync databases to their most possible abbreviated forms (sync_config => sc) before release.

Reply:
What I was saying is it's unusual that the oid by itself made the collection name too long. I wonder if there's something weird that's making the limit smaller than it should be. What's the exact error you get when leave the db and coll name as the object ID? sync_config_5b75c94c81941e2ad865a475.5b75c94c81941e2ad865a475 is only 62 bytes and the limit should be 120